privacyidea/lib/tokens/webauthntoken.py

```diff
@@ -798,13 +797,16 @@ def update(self, param, reset_failcount=True):
         transaction_id = getParam(param, "transaction_id", optional)
         reg_data = getParam(param, "regdata", optional)
         client_data = getParam(param, "clientdata", optional)
-
-        if reg_data and client_data:
-            self.init_step = 2
-
+        automatic_description = DEFAULT_DESCRIPTION
```

Comment:
same as above, `automatic_description` will be overridden in line 888

Reply:
Will check and do.

Reply:
This will only be overwritten in line 888, 

    if web_authn_credential.attestation_cert

exist. If line 870 is false, it would be missing.
So I have to keep it here.

Reply:
You are right, missed that.